src/canhydro/CylinderCollection.py

```diff
@@ -42,9 +47,24 @@
     in_flow_grade_lim=config['config_vars']["in_flow_grade_lim"]
 
 NAME = "CylinderCollection"
+def pickle_collection(collection, designation: str = ""):   
+    # file_path = "".join([output_dir, "pickle\\", f'{collection.file_name}_pickle'])
+    if designation == "": designation = collection.file_names
+    file_path ="".join(['/code/code/canopyHydrodynamics/data/output/pickle/', f'{collection.file_name.replace(".csv","")}_pickle_{designation}'])
+    directory = os.path.dirname(file_path)
+    create_dir_and_file(directory)
+    pickle_file = open(file_path, 'ab')
+    pickle.dump(collection, pickle_file) 
+    pickle_file.close()
+    return pickle_file
+
+def unpickle_collection(file_name:str):
+    file_path ="".join(['/code/code/canopyHydrodynamics/data/output/pickle/', file_name])
+    dbfile = open(file_path, 'rb')    
+    db = pickle.load(dbfile)
+    dbfile.close()
+    return db
 
-
-# By inheriting the odel class, lambda cyl : cyl.branch_order = br CC gains managed functionality- like lambda searching
```

Comment:
This class no longer inherits from the [m]odel class, and has not done so for some time